config_example.sh

```diff
@@ -1,287 +1,545 @@
+#
+#export IGNITION_EXTRA=extra.ign
+
+# ASSETS_EXTRA_FOLDER -
+# Folder where to copy extra manifests for the cluster deployment
+#
+#export ASSETS_EXTRA_FOLDER=local_file_path
 
-# Apply extra workers automatically if NUM_EXTRA_WORKERS is set
-#export APPLY_EXTRA_WORKERS=true
+# FIPS_MODE
+# Enable FIPS mode
+#
+#export FIPS_MODE=true
 
-# Indicate the online status of the NUM_EXTRA_WORKERS set in extra_host_manifests.yaml
-#export EXTRA_WORKERS_ONLINE_STATUS=true
+# DISABLE_MULTICAST -
+# In order to test using unicast for keepalived, one needs to disable multicast.
+# Setting this variable to true will block multicast via ebtables for both IPv4 
+# and IPv6.
+#
+#export DISABLE_MULTICAST=false
+
+# ENABLE_METALLB_MODE -
+# Enable metallb
+# Default is unset.
+# Choices: "l2", "bgp"
+#
+#export ENABLE_METALLB_MODE="l2"
+#
+# metallb container location (optional)
+#export METALLB_IMAGE_BASE=
+#export METALLB_IMAGE_TAG=
 
-# Provisioning interface on the helper ndoe
+################################################################################
+## Network Settings
+##
+
+# IP_STACK - IP stack for the cluster.
+# Default: "v6"
+# Choices: "v4", "v6", "v4v6"
+#
+#export IP_STACK=v4
+
+# HOST_IP_STACK - IP stack for the hosts. 
+# Default: If unset, defaults to ${IP_STACK}.
+# Choices: "v4", "v6", "v4v6"
+# You can set IP_STACK to "v4" or "v6" and HOST_IP_STACK to "v4v6" to install a 
+# single-stack cluster on dual-stack hosts.
+#
+#export HOST_IP_STACK=v4v6
+
+# NTP_SERVERS - Configure custom ntp servers if needed
+# Default is unset.
+#
+#export NTP_SERVERS="00.my.internal.ntp.server.com;01.other.ntp.server.com"
+
+# PROVISIONING_NETWORK_PROFILE - Provisioning network profile
+# Default is "Managed".
+#export PROVISIONING_NETWORK_PROFILE=Managed
+
+# Provisioning interface on the helper node
 #export PRO_IF="eno1"
 
 # Internal interface
 #export INT_IF="eno2"
 
-# Provisioning interface within the cluster
+# Provisioning interface within the cluster (Default: "enp1s0")
 #export CLUSTER_PRO_IF="eno1"
 
-# Which disk to deploy
-#export ROOT_DISK_NAME="/dev/sda"
-
-# Cluster name
-#export CLUSTER_NAME="mycluster"
-
-# Domain name
-#export BASE_DOMAIN="kni.lab.metal3.io"
-
 # Manage bridge
 #export MANAGE_BR_BRIDGE=n
 
-# Path to the json files with ipmi credentials
-#export NODES_FILE="/root/git/dev-scripts/bm.json"
+# Enable virtualMediaViaExternalNetwork in provisioning configuration.
+# More info: https://github.com/openshift/cluster-baremetal-operator/blob/master/api/v1alpha1/provisioning_types.go#L163-L171
+#export ENABLE_VIRTUAL_MEDIA_VIA_EXTERNAL_NETWORK=true
 
-# Whether the installation is on baremetal or not
-#export NODES_PLATFORM=BM
+# Set the DHCP lease time (Default: 60)
+#export DHCP_LEASE_EXPIRY=60
 
-# DNS_VIP
+# DNS_VIP (Defaults to v6)
+#
+# v4 Default:
 #export DNS_VIP="11.0.0.2"
+#
+# v6 Default:
+#export DNS_VIP="fd2e:6f44:5dd8:c956::2"
 
-# Network type
+# Network type (Defaults to v6)
```

Comment:
The `defaults to v6` here and elsewhere is slightly misleading - the values for `NETWORK_TYPE`, and subnet/VIP values are [defaulted depending on the value](https://github.com/openshift-metal3/dev-scripts/blob/master/network.sh#L104..L130) of `IP_STACK`